cypress/fixtures/todos/taskTA.json

```diff
@@ -0,0 +1,95 @@
+{
+  "tasks": [
+    {
+      "enabled": true,
```

Comment:
Is it really possible to have 2 enabled tasks? (This one and the one on line 51.)

Reply:
Mihai was very clear about this - they can't block each other

Reply:
OK!